### ISSUE TITLE
Moving a manoeuver node forward too fast crashes the game

### ISSUE THREAD

**Issue by MiracleMagician** [mockingbirdnest/Principia#2188]
![screenshot](https://user-images.githubusercontent.com/50024118/58748879-21a43c00-84b1-11e9-856c-8fc2e8201554.png)

I misclicked the place indicated in the screenshot when there's a short flight plan, and the game just crashed. Guess it's caused by manoeuver node moving beyond flight plan limit?
It's reproducedable by me, I clicked there again and it generated identical error message.

```
Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
    @   00007FF9183490A4  	principia::ksp_plugin::internal_man?uvre::Man?uvre<principia::geometry::internal_frame::Frame<enum principia::serialization::Frame_PluginTag,7,1>,principia::geometry::internal_frame::Frame<enum principia::serialization::Frame_PluginTag,4,0> >::FrenetFrame [0x00007FF9183490A3+323] (d:\principiabuild\principia\ksp_plugin\man?uvre_body.hpp:209)
    @   00007FF918418058  	principia::ksp_plugin::internal_renderer::Renderer::FrenetToWorld [0x00007FF918418057+55] (d:\principiabuild\principia\ksp_plugin\renderer.cpp:188)
    @   00007FF9183C1C0E  	principia__FlightPlanGetManoeuvreFrenetTrihedron [0x00007FF9183C1C0D+445] (d:\principiabuild\principia\ksp_plugin\interface_flight_plan.cpp:309)
    @   00000000262C3934  	(No symbol) [0x00000000262C3933]
F0601 20:59:49.051201  8440 man?uvre_body.hpp:209] Check failed: it != coasting_trajectory_->End()
```

**Comment by MiracleMagician** [comment]
Game version 1.7.0, Principia Version Fatou (Yeah I downloaded the latest released code and compiled it by myself... Can't wait for the new moon)

**Comment by eggrobin** [comment]
Can you provide us with the `INFO` file that ends with that stack trace?

**Comment by MiracleMagician** [comment]
This is the INFO file at the second crash, so hopefully it will be clean enough.
[INFO.20190601-210342.884.txt](https://github.com/mockingbirdnest/Principia/files/3243799/INFO.20190601-210342.884.txt)

**Comment by eggrobin** [comment]
I am unable to reproduce the issue in Fatou (pulling on the initial time just keeps the manœuvre at the end of the flight plan, but does not cause a crash).

What is the exact commit number you used?

Normally this is given in the INFO log (it comes from version.hpp, generated by [generate_version_translation_unit](https://github.com/mockingbirdnest/Principia/blob/2019060310-Fatou/generate_version_translation_unit.ps1)), but yours somehow seems to be missing that information, and to have a nonsensical commit date as well:
```
I0601 21:03:42.900367  3892 interface.cpp:629] Principia version  built on 0001-01-01T00:00:00Z by Microsoft Visual C++ version 191627031 for Windows x86-64
```

**Comment by MiracleMagician** [comment]
I downloaded the version in release page, checked that all google dependencies are up to date, linked ksp's dll files in plugin adapter and simply clicked build. I noticed there were errors while compiling, maybe it's a problem with my visual studio or that my computer using a non-English language? Since I've noticed that before ui overhaul in fano, Principia built on my computer's version information in main panel was in garbled Chinese, and after fano it just simply doesn't show up.
